.changeset/rotten-jobs-fix.md

```diff
@@ -0,0 +1,5 @@
+---
+"@ledgerhq/live-common": minor
```

Comment:
should be patch ^

Reply:
I don't think so, it's not patching a bug or a feature, it's bringing to the outside world (metamask & co) a new functionality while preserving retrocompatibility. 🤔

Reply:
Hmm not really sure about that, I'm not really "fixing" something as the problem isn't coming from us, but I'm completely changing the gas estimation functionnality while being backwards compatible, so it feels more like a minor than a patch to me 🤔 But I can change that if you really feel like it's problematic (although it's a minor in the last LLM so...)

Reply:
since it goes into a hotfix, we definitely want a patch as it will fuck up the versioning communicated by marketing team already :trollface:

Reply:
I personally did not check the package, my bad, minor on live-common is good